docs/examples.rst

```diff
@@ -853,7 +853,46 @@ exactly is happening during test execution. This also allows to
 inspect particular place of the code by inserting a ``bash`` in
 the shell code or ``import pdb; pdb.set_trace()`` for python::
 
-    tmt run --all execute --how beakerlib.tmt --interactive
+    tmt run --all execute --how tmt --interactive
+
+
+Aliases
+------------------------------------------------------------------
+
+It might be useful to set up a set of shell aliases for the tmt
+command lines which you often use. For a quick reservation of
+a machine or a container for quick experimenting::
+
+    alias reserve='tmt run login --step execute execute finish provision --how container --image fedora'
```

Comment:
Why `provision --how container --image fedora` would be default, instead of tmt's default? But I guess it's just an example (the one ppl will just copy-paste).

Reply:
I used this as an example alias becuase containers are fast. It's up to the user to adjust as/if needed.